apps/picsa-tools/option-tool/src/app/components/editor/editor.component.ts

```diff
@@ -32,15 +34,15 @@ export class EditorComponent implements OnInit {
   ]
   this.perfomanceValues = { lowRf:"ok",midRf:"ok",highRf:"ok"}
   this.investmentValues = {money:'high', time:'high'}
+  this.warningText='';
```

Comment:
nit(non-blocking)
As these values are hardcoded you can also just provide the default values when you declare the variable above, i.e. 
```
export class EditorComponent implements OnInit {
  // stepCounter: number;
  warningText = ''
```
The benefit of this is less duplication, and you do not need to explicitly define the type as it will be inferred from the deafult value (will assume string)